host/src/lib/HelperFunctions.jsx

```diff
@@ -174,10 +139,180 @@ export const getAnswersByQuestion = (
               isCorrect: false,
             });
           }
+          answersArray[
+            choicesTextArray.indexOf(choice.text)
+          ].count += 1;
+          answersArray[choicesTextArray.indexOf(choice.text)].teams.push(team.name);
         }
       });
     });
     return { answersArray, confidenceArray };
   }
   return { answersArray: [], confidenceArray };
 };
+
+export const getShortAnswers = (shortAnswerResponses) => {
+  // create this to use as an index reference for the confidence levels to avoid find/findIndex
+  const confidenceLevelsArray = Object.values(ConfidenceLevel);
+  let confidenceArray = createBlankConfidenceArray();
+  if (shortAnswerResponses && shortAnswerResponses.length > 0) {
+    shortAnswerResponses.forEach((answer) => {
+      if (answer.teams){
+        answer.teams.forEach((team) => {
+          console.log(team);
+          const index = confidenceLevelsArray.indexOf(team.confidence);
+          if (answer.isCorrect) {
+            confidenceArray[index].correct += 1;
+            confidenceArray[index].players.push({
+              name: team.team,
+              answer: answer.value,
+              isCorrect: true,
+            });
+          } else {
+            confidenceArray[index].incorrect += 1;
+            confidenceArray[index].players.push({
+              name: team.team,
+              answer: answer.value,
+              isCorrect: false,
+            });
+          }
+        });
+      }
+    });
+    return { answersArray: shortAnswerResponses, confidenceArray};
+  };
+  return { answersArray: [], confidenceArray };
+};
+
+export const determineAnswerType = (answer) => {
```

Comment:
these are the equality check functions. for more information see:
https://github.com/rightoneducation/righton-app/wiki/Short-Answer-Response-%E2%80%90-Equality-Checks

Reply:
This has been updated per: 
[[Host/Play/Networking] - Short Answer Response - Adjusts Answer Handling #858](https://github.com/rightoneducation/righton-app/pull/858)